### PR TITLE
Remove containing type from lightbulb for AddParameter and GenerateMember

### LINKED ISSUE #56485: Can't read lightbulb entries when they are too long
image

HOvering over the items does not show the full title either.  This may need to go to the editor for discussion.

### PR DESCRIPTION
**Pull request by akhera99**
Fixes #56485

There still might be code actions that have the fully qualified type; I need to look through the code actions to see which do. These fix the ones from the issue filed.
Need to reach out to editor team to add hover over code action to display the full string if it's cut off.

### CODE REVIEW

File: src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.CodeAction.cs [hunk 1]
```diff
@@ -52,8 +52,7 @@ private partial class GenerateParameterizedMemberCodeAction : CodeAction
                             isAbstract ? FeaturesResources.Generate_abstract_method_1_0 : FeaturesResources.Generate_method_1_0;
 
                         var name = state.IdentifierToken.ValueText;
-                        var destination = state.TypeToGenerateIn.Name;
-                        return string.Format(text, name, destination);
+                        return string.Format(text, name);
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.CodeAction.cs:55]: this seems broken.  the resource strings have 1_0 in them, indicating they take two args.  But you're only passing in one.

File: src/Features/Core/Portable/GenerateMember/GenerateEnumMember/AbstractGenerateEnumMemberService.CodeAction.cs [hunk 2]
```diff
@@ -60,12 +60,10 @@ public override string Title
             {
                 get
                 {
-                    var text = FeaturesResources.Generate_enum_member_1_0;
+                    var text = FeaturesResources.Generate_enum_member_0;
 
                     return string.Format(
```
Comment (ryzngard) [inline src/Features/Core/Portable/GenerateMember/GenerateEnumMember/AbstractGenerateEnumMemberService.CodeAction.cs:65]: nit: put this on one line

**Review by CyrusNajmabadi — APPROVED** [review]

File: src/Features/Core/Portable/FeaturesResources.resx [hunk 3]
```diff
@@ -225,17 +225,17 @@
...
+  <data name="Generate_enum_member_0" xml:space="preserve">
+    <value>Generate enum member '{0}'</value>
   </data>
   <data name="Generate_constant_1_0" xml:space="preserve">
     <value>Generate constant '{1}.{0}'</value>
   </data>
   <data name="Generate_read_only_property_1_0" xml:space="preserve">
     <value>Generate read-only property '{1}.{0}'</value>
   </data>
-  <data name="Generate_property_1_0" xml:space="preserve">
```
Comment (ryzngard) [inline src/Features/Core/Portable/FeaturesResources.resx:237]: xlf files need to be generated

**Review by ryzngard — APPROVED** [review]

File: src/Features/Core/Portable/FeaturesResources.resx (hunk 3 above)
Comment (akhera99) [reply src/Features/Core/Portable/FeaturesResources.resx:237]: yeah, my VS is having issues freezing right now so I needed to restart

**Pull request merged**